infra/horreum-infra-common/src/main/resources/init-script.sql

```diff
@@ -0,0 +1,2 @@
+-- TODO: this file is does not have any effect at the moment
```

Comment:
can you remove this empty file and open an issue for this?

Reply:
Added #1163 to track that feature